src/client/components/Button.tsx

```diff
@@ -21,10 +21,17 @@ const Button: FunctionComponent<ButtonProps> = ({
   sx,
   onClick,
   disabled,
+  darkMode,
 }) => {
+
   const isPrimary = variant === 'primary';
   const isSecondary = variant === 'secondary';
+  const primaryBackGroundColor = !darkMode ? '#E11774' : '#0AFFF4';
```

Comment:
Along with the mobile responsiveness, I am updating the theme used by MUI components so that our theme color information is stored in one place vs in individual components. Happy to explain how it works when it's ready. This works for the time being but wanted everyone to be aware.